src/docfx/lib/markdown/MarkdigUtility.cs

```diff
@@ -17,7 +17,10 @@ public static SourceInfo ToSourceInfo(this MarkdownObject obj, int? line = null,
         {
             // Line info in markdown object is zero based, turn it into one based.
             if (obj != null)
-                return new SourceInfo(file?.FilePath ?? (InclusionContext.File as Document)?.FilePath, obj.Line + 1, obj.Column + columnOffset + 1);
+            {
+                return new SourceInfo(
```

Comment:
see if you can extract a local variable
```suggestion
                var file = file?.FilePath ?? (InclusionContext.File as Document)?.FilePath;
                return new SourceInfo(file, ...)
```